packages/server/__snapshots__/2_browser_path_spec.coffee.js

```diff
@@ -1,3 +1,14 @@
+exports['e2e launching browsers by path fails with bad browser path 1'] = `
+We could not identify a known browser at the path you provided: \`/this/aint/gonna/be/found\`
+
+The output from the command we ran was:
+
+ Command failed: /bin/sh -c "/this/aint/gonna/be/found" --version
+/bin/sh: 1: /this/aint/gonna/be/found: not found
+
+
+`
+
```

Comment:
@flotwig I updated this snapshot - please review and ensure it's correct.

Reply:
I thought we deleted that snapshot in #4009, this shouldn't be needed anymore.